src/components/PastWorkOrders/MobileWorkingPastWorkOrdersView/MobileWorkingPastWorkOrdersView.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState, useEffect, useRef } from 'react'
+import { frontEndApiRequest } from '@/utils/frontEndApiClient/requests'
+import { beginningOfDay } from '@/utils/time'
+
+import PastWorkOrdersDatePicker from '../../PastWorkOrdersDatePicker/Index'
+import Spinner from '../../Spinner'
+import ErrorMessage from '../../Errors/ErrorMessage'
+import WarningInfoBox from '../../Template/WarningInfoBox'
+import Meta from '../../Meta'
+import { WorkOrder } from '../../../models/workOrder'
+import { MobileWorkingPastWorkOrderListItems } from './MobileWorkingPastWorkOrderListItems'
+import {
+  CurrentUser,
+  WorkOrderType,
+  WorkOrdersType,
+} from '@/root/src/types/variations/types'
+
+const SIXTY_SECONDS = 60 * 1000
+
+const MobileWorkingPastWorkOrdersView = ({ currentUser }) => {
+  const currentDate = beginningOfDay(new Date())
+  const yesterday = new Date()
+  yesterday.setDate(currentDate.getDate() - 1)
+
+  const [visitedWorkOrders, setVisitedWorkOrders] = useState(null)
+  const [sortedWorkOrders, setSortedWorkOrders] = useState(null)
+  const [error, setError] = useState<string | null>()
+  const [selectedDate, setSelectedDate] = useState<Date>(yesterday)
+  const targetDate = selectedDate.toISOString().split('T')[0]
+
+  const getOperativeWorkOrderView = async () => {
+    setError(null)
+
+    try {
+      const data = await frontEndApiRequest({
+        method: 'get',
+        path: `/api/operatives/${currentUser.operativePayrollNumber}/workOrdersNew?date=${targetDate}`,
+      })
+
+      const workOrders: WorkOrdersType = data.map(
+        (wo: WorkOrderType) => new WorkOrder(wo)
+      )
+
+      const visitedWorkOrders = workOrders.filter((wo: WorkOrderType) =>
+        wo.hasBeenVisited()
+      )
+
+      const sortedWorkOrderItems = sortWorkOrderItems(currentUser, workOrders)
+      setVisitedWorkOrders(visitedWorkOrders)
+      setSortedWorkOrders(sortedWorkOrderItems)
+    } catch (e) {
+      setVisitedWorkOrders(null)
+      setSortedWorkOrders(null)
+      console.error('An error has occured:', e)
+      setError(
+        `Request failed with status code: ${e.response?.status} with message: ${e.response?.data?.message}`
+      )
+    }
+  }
+
+  const intervalRef = useRef(null)
+
+  useEffect(() => {
+    // dont try fetch if currentUser not loaded yet
+    if (currentUser === null) return
+
+    // initial fetch (otherwise it wont fetch until interval)
+    getOperativeWorkOrderView()
+
+    const intervalId = setInterval(() => {
+      getOperativeWorkOrderView()
+    }, SIXTY_SECONDS)
```

Comment:
what do we need this for?

Reply:
might not need this use case - remove but check it doesn't break anything first